plugins/workspace-minimap/test/coordinate_conversion.mocha.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+const chai = require('chai');
+const assert = chai.assert;
+const {Minimap} = require('../src/minimap');
+
+
```

Comment:
It looks like each suite is broadly testing the same thing, just with different relative sizes; could you factor the variable bits (metrics, click positions, expected results) into an array, and then just loop through that and move the suite/test calls inside?

Reply:
I would actually recommend against doing this! While it works great with most test systems, I don't think it works well with Mocha, because there's no good way to isolate tests after you've separated the data out into a different structure.

I prefer using the AAA structure, even in cases where you have basic inputs/outputs.

But if you have strong opinions Aaron, happy to defer.

Reply:
Beka raised a good point offline about not being able to .only/.skip the tests if they're structured this way, and she has a lot more experience with Mocha so I'm happy to defer to her. The existing test structure LGTM, sorry for the tangent here!